libs/features/mas/src/lana.js

```diff
@@ -1,58 +1,61 @@
-const Defaults = {
+// Default configuration for logging system
+const config = {
     clientId: 'merch-at-scale',
     delimiter: '¶',
     ignoredProperties: ['analytics', 'literals'],
     serializableTypes: ['Array', 'Object'],
-    // Sample rate is set to 100 meaning each error will get logged in Splunk
-    sampleRate: 30,
-    tags: 'consumer=milo/commerce',
+    sampleRate: 1,
+    tags: 'consumer,acom,mas',
+    isProdDomain: false,
 };
 
 const seenPayloads = new Set();
 
-const isError = (value) =>
-    value instanceof Error ||
-    // WCS error response
-    // TODO: check if still actual
-    typeof value.originatingRequest === 'string';
+function isError(value) {
+    return (
+        value instanceof Error || typeof value?.originatingRequest === 'string'
+    );
+}
 
 function serializeValue(value) {
     if (value == null) return undefined;
     const type = typeof value;
+
     if (type === 'function') {
-        const { name } = value;
-        return name ? `${type} ${name}` : type;
+        return value.name ? `function ${value.name}` : 'function';
     }
-    if (type === 'object') {  
+
+    if (type === 'object') {
         if (value instanceof Error) return value.message;
+
         if (typeof value.originatingRequest === 'string') {
-          /* c8 ignore next 4 */
             const { message, originatingRequest, status } = value;
             return [message, status, originatingRequest]
-                .filter((v) => v)
+                .filter(Boolean)
                 .join(' ');
         }
-        const name =
-            value[Symbol.toStringTag] ??
+
+        const objectType =
+            value[Symbol.toStringTag] ||
```

Comment:
I think we should check against null objects also to prevent runtime errors:
```suggestion
const objectType =
    value[Symbol.toStringTag] ||
    (Object.getPrototypeOf(value)?.constructor?.name || '');
```

Reply:
actually I revert my change from `??` to `||` that was unintentional.